modules/rhel-compute-requirements.adoc

```diff
@@ -51,6 +51,12 @@ be configured according to its
 link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html[storage guidelines]
 and the `disk.enableUUID=true` attribute must be set.
 
+* Each system should be able to access the cluster's api
```

Comment:
Acronyms should be all uppercase: API, DNS, URL.

Reply:
Thank you.